karaf/karaf-features/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The MIT License
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>influxdb-karaf</artifactId>
+        <groupId>com.influxdb</groupId>
+        <version>2.2.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>influxdb-karaf-features</artifactId>
+    <packaging>feature</packaging>
+
+    <name>Apache Karaf Features for InfluxDB 2.0</name>
+    <description>
+        Apache Karaf Features for InfluxDB 2.0: feature definition of client bundles and dependencies.
+    </description>
+
+    <properties>
+        <dependency.gson-fire.version>1.8.3</dependency.gson-fire.version>
+        <dependency.reactivestreams.version>1.0.3</dependency.reactivestreams.version>
+    </properties>
```

Comment:
Is this necessary?

Reply:
Separete bundles/JARs (with minimum required versions) were added to feature definition in first approach that is replaced by embedding dependencies to client-osgi JAR (bundle). It is not necessary anymore.